p0_user_stories.md

```diff
@@ -6,13 +6,13 @@ Taken from https://docs.google.com/document/d/10t4q5XO1ED2PnK3ishn4y3G4Tma7uMYge
 
 ## tier-1
 
-- I want 2 apps in different namespaces to connect, but can't read (or write) the labels for those namespaces from my Kubernetes client.   I cannot select pods for this because I don't know all the labels and names of pod/services that i want to contact ahead of time. 
+- I want 2 apps in different namespaces to connect, but can't read (or write) the labels for those namespaces from my Kubernetes client.   I cannot select pods for this because I don't know all the labels and names of pod/services that I want to contact ahead of time. 
```

Comment:
This User story is related to using a namespace name (instead of label) for selector. The situation here, for me is:

* I want to allow all the applications from namespace-b to connect to my workloads here in namespace-a
* I don't know what are the labels of namespace-b
* I don't know what are the labels of the Pods in namespace-b